### PR TITLE
BRC Tawnos Solemn Survivor

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Types:Legendary Creature Human Artificer
 PT:1/3
 A:AB$ CopyPermanent | Cost$ 2 T | ValidTgts$ Artifact.YouCtrl+token | TgtPrompt$ Select target artifact token you control | SubAbility$ DBMill | SpellDescription$ Create a token that's a copy of up to one target artifact token you control. Mill two cards.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt:5]: Up to one needs TgtMin (update Prompt too)

File: forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ Types:Legendary Creature Human Artificer
...
 SVar:DBMill:DB$ Mill | Mill | NumCards$ 2
-A:AB$ CopyPermanent | Cost$ 1 W U B T ExileFromGrave<1/Artifact;Creature/artifact or creature card> Sac<2/Artifact.token/artifact tokens> | Defined$ Exiled | AddTypes$ Artifact | SpellDescription$ Create a token that's a copy of the exiled card, except it's an artifact in addition to its other types. Activate only as a sorcery.
+A:AB$ CopyPermanent | Cost$ 1 W U B T ExileFromGrave<1/Artifact;Creature/artifact or creature card> Sac<2/Artifact.token/artifact tokens> | Defined$ Exiled | AddTypes$ Artifact | SorcerySpeed$ True | SpellDescription$ Create a token that's a copy of the exiled card, except it's an artifact in addition to its other types. Activate only as a sorcery.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt:7]: Make cost order same as Oracle

File: forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Tawnos, Solemn Survivor
+ManaCost:1 U
+Types:Legendary Creature Human Artificer
+PT:1/3
+A:AB$ CopyPermanent | Cost$ 2 T | ValidTgts$ Artifact.YouCtrl+token | TgtPrompt$ Select target artifact token you control | SubAbility$ DBMill | SpellDescription$ Create a token that's a copy of up to one target artifact token you control. Mill two cards.
+SVar:DBMill:DB$ Mill | Mill | NumCards$ 2
+A:AB$ CopyPermanent | Cost$ 1 W U B T ExileFromGrave<1/Artifact;Creature/artifact or creature card> Sac<2/Artifact.token/artifact tokens> | Defined$ Exiled | AddTypes$ Artifact | SorcerySpeed$ True | SpellDescription$ Create a token that's a copy of the exiled card, except it's an artifact in addition to its other types. Activate only as a sorcery.
+DeckHints:Ability$Graveyard|Mill & Type$ Artifact
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt:8]: `Type$ Artifact` > `Type$Artifact`

File: forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt (hunk 2 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt:7]: what you mean ?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt:7]: Nvm i see now

File: forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Tawnos, Solemn Survivor
+ManaCost:1 U
+Types:Legendary Creature Human Artificer
+PT:1/3
+A:AB$ CopyPermanent | Cost$ 2 T | ValidTgts$ Artifact.YouCtrl+token | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target artifact token you control | SubAbility$ DBMill | SpellDescription$ Create a token that's a copy of up to one target artifact token you control. Mill two cards.
+SVar:DBMill:DB$ Mill | Mill | NumCards$ 2
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt:6]: ` | Mill` not needed

File: forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Tawnos, Solemn Survivor
+ManaCost:1 U
+Types:Legendary Creature Human Artificer
+PT:1/3
+A:AB$ CopyPermanent | Cost$ 2 T | ValidTgts$ Artifact.YouCtrl+token | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target artifact token you control | SubAbility$ DBMill | SpellDescription$ Create a token that's a copy of up to one target artifact token you control. Mill two cards.
+SVar:DBMill:DB$ Mill | NumCards$ 2
+A:AB$ CopyPermanent | Cost$ 1 W U B T Sac<2/Artifact.token/artifact tokens> ExileFromGrave<1/Artifact;Creature/artifact or creature card> | Defined$ Exiled | AddTypes$ Artifact | SorcerySpeed$ True | SpellDescription$ Create a token that's a copy of the exiled card, except it's an artifact in addition to its other types. Activate only as a sorcery.
+DeckHints:Ability$Graveyard|Mill & Type$Artifact 
+DeckHas:Ability$Mill|Graveyard|Token|Sacrifice|Token & Type$Artifact
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tawnos_solemn_survivor.txt:9]: Token twice

**Review by tool4ever — APPROVED** [review]

**Pull request merged**